frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -16,24 +16,32 @@ import { VMKind, VMIKind } from '../../types';
 import { VMTemplateLink } from '../vm-templates/vm-template-link';
 import { getBasicID, prefixedID } from '../../utils';
 import { vmDescriptionModal, vmFlavorModal } from '../modals';
+import { VMCDRomModal } from '../modals/cdrom-vm-modal';
 import { getDescription } from '../../selectors/selectors';
+import { getCDRoms } from '../../selectors/vm/selectors';
 import { getVMStatus } from '../../statuses/vm/vm';
 import { getFlavorText } from '../flavor-text';
 import { EditButton } from '../edit-button';
 import { getVmiIpAddressesString } from '../ip-addresses';
+import { DiskSummary } from '../vm-disks/disk-summary';
 
 import './_vm-resource.scss';
```

Comment:
Please remove the underscode

Reply:
Let's not miss that comment :-)

Reply:
this is not my code. is it in the scope of this PR? @mareklibra